unlock-app/src/components/interface/locks/Settings/elements/SettingEmail.tsx

```diff
@@ -19,7 +19,7 @@ export const SettingEmail = ({
   return (
     <div className="grid grid-cols-1 gap-6">
       <SettingCard
-        label="Key Purchased template"
+        label="Key Purchased Template"
```

Comment:
I would use `capitalize` as class on the component where you need text to always capitalise. Seems like it is the case for all collapsible menus?

Reply:
i think is a good idea! just added